src/net_processing.cpp

```diff
@@ -1518,7 +1518,8 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         }
 
         CBlock block;
-        assert(ReadBlockFromDisk(block, it->second, chainparams.GetConsensus()));
+        bool ret = ReadBlockFromDisk(block, it->second, chainparams.GetConsensus());
```

Comment:
Was introduced in https://github.com/bitcoin/bitcoin/commit/da9cdd2c9cb58e3737b727c21f070f074b1247a7 by @TheBlueMatt